lab-09-2-xor-nn.py

```diff
@@ -44,14 +33,14 @@
     sess.run(tf.global_variables_initializer())
 
     for step in range(10001):
-        sess.run(train, feed_dict={X: x_data, Y: y_data})
+        _, cost_val = sess.run([train, cost], feed_dict={X: x_data, Y: y_data})
         if step % 100 == 0:
-            print(step, sess.run(cost, feed_dict={
-                  X: x_data, Y: y_data}), sess.run([W1, W2]))
+            print(step, cost_val)
 
     # Accuracy report
-    h, c, a = sess.run([hypothesis, predicted, accuracy],
-                       feed_dict={X: x_data, Y: y_data})
+    h, c, a = sess.run(
+        [hypothesis, predicted, accuracy], feed_dict={X: x_data, Y: y_data}
+    )
     print("\nHypothesis: ", h, "\nCorrect: ", c, "\nAccuracy: ", a)
```

Comment:
Can you change to **f-string** and also change **Correct** to **Predicted**?

```suggestion
     print(f"""
Hypothesis: 
{h}
Predicted: 
{p}
Accuracy: {a}""")
```